test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I like that the test for one user tests the details of the user, whereas the test for two users just tests the count.